### PR TITLE
fix: condition got error when cowOrders was undefined

### PR DESCRIPTION
**Pull request by Diogomartf**
**Description**
Fix error when opening a modal with cowOrders undefined

**Summary**
- add it to calculateStackAveragePrice and totalStackOrdersDone

**Error**
image

### CODE REVIEW

File: packages/app/models/stack-order/stack-order.ts [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
+
+  return order.cowOrders.length;
+};
 
 export const calculateStackAveragePrice = (order: StackOrder) => {
   let totalExecutedBuyAmount = 0;
   let totalExecutedSellAmount = 0;
 
-  if (order.cowOrders.length === 0) return 0;
+  if (noCowOrders(order)) return 0;
```
Comment (berteotti) [inline packages/app/models/stack-order/stack-order.ts:17]: This looks simpler to me

```suggestion
  if (!order?.cowOrders?.length) return 0;
```

**Review by ElRodrigote — DISMISSED** [review]
I agree with Leo's comment, but I'll leave my approve as it is a small enough fix either way

File: packages/app/models/stack-order/stack-order.ts (hunk 1 above)
Comment (Diogomartf) [reply packages/app/models/stack-order/stack-order.ts:17]: I'll test it, it it works, I'm down with this

Comment (Diogomartf) [reply packages/app/models/stack-order/stack-order.ts]: Fixed!

**Review by berteotti — APPROVED** [review]

**Pull request merged**